### PR TITLE
BROKER: Insert Candidate

### PR DESCRIPTION
**Pull request by imjane5**

### CODE REVIEW

File: CandidateApp.ConsoleApp/Brokers/StorageBrokers/IStorageBroker.Candidate.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// ------------------------------------------------
+
+using CandidateApp.ConsoleApp.Models.Candidate;
+
+namespace CandidateApp.ConsoleApp.Brokers.StorageBrokers
+{
+    partial interface IStorageBroker
+    {
+        Candidate InsertCandidate(Candidate candidate);
+
```
Comment (ShriHumrudha) [inline CandidateApp.ConsoleApp/Brokers/StorageBrokers/IStorageBroker.Candidate.cs]: Remove this extra line

File: CandidateApp.ConsoleApp/Brokers/StorageBrokers/StorageBroker.Candidate.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+using System.Collections.Generic;
+using CandidateApp.ConsoleApp.Models.Candidate;
+
+namespace CandidateApp.ConsoleApp.Brokers.StorageBrokers
+{
+    partial class StorageBroker : IStorageBroker
+    {
+        List<Candidate> Candidates = new List<Candidate>();
+        public Candidate InsertCandidate(Candidate candidate)
```
Comment (ShriHumrudha) [inline CandidateApp.ConsoleApp/Brokers/StorageBrokers/StorageBroker.Candidate.cs:14]: We need a line break between single and multi-line code.

File: CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CandidateApp.ConsoleApp.Models.Candidate
+{
+    class Candidate
+    {
+        public int CID { get; set; }
```
Comment (ShriHumrudha) [inline CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs]: What is this CID?

File: CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CandidateApp.ConsoleApp.Models.Candidate
```
Comment (ShriHumrudha) [inline CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs]: The folder should be in plural - Candidates

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CandidateApp.ConsoleApp/Brokers/StorageBrokers/StorageBroker.Candidate.cs (hunk 2 above)
Comment (imjane5) [reply CandidateApp.ConsoleApp/Brokers/StorageBrokers/StorageBroker.Candidate.cs:14]: Added an extra line

File: CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs (hunk 3 above)
Comment (imjane5) [reply CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs]: Changed as CandidateID

File: CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs (hunk 4 above)
Comment (imjane5) [reply CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs]: Changed the folder name as well

File: CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs (hunk 3 above)
Comment (ShriHumrudha) [reply CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs]: Similar to FirstName and LastName you can just say 'Id' as we are in the context of Candidate we need not repeat Candidate in the attributes too.

File: CandidateApp.ConsoleApp/Models/Candidates/Candidate.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CandidateApp.ConsoleApp.Models.Candidate
+{
+    class Candidate
+    {
+        public int ID { get; set; }
```
Comment (ShriHumrudha) [inline CandidateApp.ConsoleApp/Models/Candidates/Candidate.cs]: Id not ID

**Review by ShriHumrudha — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**